### PR TITLE
feat: Community Announcements Page

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Preview [link](https://near.org/megha19.near/widget/app?page=community&handle=devhub-platform)

### CODE REVIEW

**Comment by Megha-Dev-19** [comment]
Am I allowed to attach the figma design as well? for reference?

File: src/devhub/entity/community/Compose.jsx [hunk 1]
```diff
@@ -0,0 +1,477 @@
+if (!context.accountId) {
+  return <></>;
+}
+
+const optimisticUpdateFn = props.optimisticUpdateFn;
+const clearOptimisticUpdateFn = props.clearOptimisticUpdateFn;
+
```
Comment (elliotBraem) [inline src/devhub/entity/community/Compose.jsx:7]: Where did these two functions come from? It looks like optimisticUpdate is for immediately reflecting the newly created post?

Comment (Megha-Dev-19) [reply src/devhub/entity/community/Compose.jsx:7]: yeah, we can use it for passing some custom function for commit and cancel as well

**Comment by ailisp** [comment]
Looks good so far! I think it is blocked by https://github.com/NEAR-DevHub/neardevhub-contract/pull/79 to call smart contract to post from community account

**Pull request merged**